### PR TITLE
man: add version information for dbus interfaces

### PR DESCRIPTION
**Pull request by abderrahim**
These only go back to version 250 which is the first version to provide the export-dbus-interfaces build target.

### CODE REVIEW

**Comment by abderrahim** [comment]
Tests are failing, marking as draft for now.

**Comment by yuwata** [comment]
```
Failed to mount /proc/sys (type n/a) on /proc/sys (MS_BIND ""): Invalid argument
```
Maybe, the recent revert broke mounting sysfs?
Anyway, not related to this PR.

**Comment by bluca** [comment]
It worked in the revert PR, so seems unlikely

**Comment by bluca** [comment]
also it's failing before the build step can start, so shouldn't our fault

**Review by bluca — APPROVED** [review]

File: man/org.freedesktop.hostname1.xml [hunk 1]
```diff
@@ -418,4 +418,16 @@ node /org/freedesktop/hostname1 {
...
   </refsect1>
+
+  <refsect1>
+    <title>History</title>
+    <refsect2>
+      <title>The D-Bus API</title>
+      <para><varname>FirmwareVersion</varname> and
+      <function>GetHardwareSerial()</function> were added in version 251.</para>
+      <para><varname>OperatingSystemSupportEnd</varname>,
+      <varname>FirmwareVendor</varname> and
```
Comment (keszybz) [inline man/org.freedesktop.hostname1.xml]: Also here, need oxford comma as in the other PR.

**Review by keszybz — CHANGES_REQUESTED** [review]

**Review by bluca — APPROVED** [review]

**Comment by Werkov** [comment]
There is the DBus introspection that gives full information about exiting APIs regardless of version. Generally, users should not rely on version numbers guaranteeing anything in API but on API itself (which is so easy with the introspection).
I don't understand the reasoning of the commit d9d2d16aeaf7c18a7b2486cd7c2db484c99050df.
@abderrahim Can you enlighten me what is the purpose? (If anything it could document history of members added _before_ `export-dbus-interfaces` was available.)

**Comment by Werkov** [comment]
The website may display the properties generated thanks to `export-dbus-interfaces` for the documented version.

**Comment by bluca** [comment]
That doesn't tell you when it was added, it just spits out what's there currently

**Comment by Werkov** [comment]
(If some seeks such an information, I'd point them to an [archive](http://web.archive.org/web/20230000000000*/https://www.freedesktop.org/software/systemd/man/latest/org.freedesktop.systemd1.html). Actually, I noticed that also the FDO docs website has own version archive).)

But the point is that such information implicitly guides users away from introspection, which is the reliable way to base API decisions on.
The less rolling/longer cycle a distro is, the greater is the discrepancy between version-based and introspection-based information.

**Comment by bluca** [comment]
To know when an interface was added

**Pull request merged**